### PR TITLE
copiousoutput HTML, qvm-open-in-dvm the rest

### PR DESCRIPTION
**Pull request by gasull**

### CODE REVIEW

**Comment by marmarek** [comment]
> Why did the Travis check fail?

Looks unrelated to your changes. It complains about word `backport` not being in dictionary, but it was introduced by totally different change.

File: configuration/mutt.md [hunk 1]
```diff
@@ -194,29 +194,7 @@ In `.urlview`:
...
-    image/jpeg; qvm-open-in-dvm %s
-    image/png; qvm-open-in-dvm %s
-    image/gif; qvm-open-in-dvm %s
-
-    # PDFs
-    application/pdf; qvm-open-in-dvm %s
-
-    # HTML
-    text/html; qvm-open-in-dvm %s
+    text/html; w3m '%s' | cat --squeeze-blank; nametemplate=%s.html; copiousoutput
```
[the remark this answers is not in the document]
Comment (gasull) [reply configuration/mutt.md]: I believe `nametemplate` solves that.  It works for me.

File: configuration/mutt.md [hunk 2]
```diff
@@ -194,29 +194,7 @@ In `.urlview`:
...
+    */*; qvm-open-in-dvm xdg-open '%s'; test=test -n "$DISPLAY"
```
[the remark this answers is not in the document]
Comment (gasull) [reply configuration/mutt.md]: It works for me in Debian.  Are you using Fedora?

File: configuration/mutt.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (gasull) [reply configuration/mutt.md]: I added `-T text/html`.  Not sure it's needed but a good idea anyway.

File: configuration/mutt.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (marmarek) [reply configuration/mutt.md]: I've checked. It doesn't work on Debian either.
```
ii  mutt           1.7.2-1+deb9 amd64        text-based mailreader supporting 
```
With mailcap with just one line:
```
*/*; qvm-open-in-dvm '%s'; test=test -n "$DISPLAY"
```
mutt prints `No matching mailcap entry found.  Viewing as text.` The same without `test=`. When I specify actual mime type(s), it works.

[the remark this answers is not in the document]
Comment (gasull) [reply configuration/mutt.md]: Removed `*/ *`.

**Comment by andrewdavidwong** [comment]
Closing this for now due to the prolonged lack of activity. If you believe this is a mistake, or if anyone would like this PR to remain open, please leave a comment, and we'll be happy to reopen this. Thank you.

**Comment by gasull** [comment]
@marmarek , would you be able to review the changes you requested?

**Comment by andrewdavidwong** [comment]
Since `mutt.md` has since been moved to External Documentation, I'm going to merge this now. Sorry about the long wait, @gasull.

**Pull request merged**